src/pages/AccountPage.js

```diff
@@ -300,6 +302,16 @@ function AccountPage({ account }) {
           >
             <PositionList positions={positions} />
           </Panel>
+          <TYPE.main fontSize={'1.125rem'} style={{ marginTop: '3rem' }}>
+            Mining Pool
```

Comment:
Small nit, but maybe call this "Liquidity Mining Pools"

Reply:
done